test/middleware/AppliedCompanyAuthMiddleware.test.ts

```diff
@@ -0,0 +1,42 @@
+import { expect } from 'chai';
+import { MOVED_TEMPORARILY } from 'http-status-codes';
+import request from 'supertest';
+import { createApp } from '../ApplicationFactory';
+
+import { Appeal } from 'app/models/Appeal';
+import { ApplicationData } from 'app/models/ApplicationData';
+import { REVIEW_PENALTY_PAGE_URI, SELECT_THE_PENALTY_PAGE_URI } from 'app/utils/Paths';
+
+
+describe('All pages after the Penalty Details page:', () => {
+
+    const pageList = [
+        ['Select Penalty', SELECT_THE_PENALTY_PAGE_URI],
```

Comment:
Data structure is bit archaic. With destructuring assignment in TS / JS it could been an array of objects which you could destruct into named arguments in the test method.

Doc: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

Reply:
Agreed, done.